.kokoro/docs.sh

```diff
@@ -30,4 +30,4 @@ BIN=./node_modules/.bin
 npm install broken-link-checker
 npm install http-server
 $BIN/http-server -p 8080 docs/ &
-$BIN/blc http://localhost:8080 -r --exclude www.googleapis.com
```

Comment:
@kinwa91 looks like a problem with what's in synthtool templates :/  Can you please take a look?